pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -138,7 +140,8 @@ stdenv.mkDerivation (finalAttrs: {
       makeWrapper ${electron}/bin/electron $out/bin/vencorddesktop \
         --prefix LD_LIBRARY_PATH : ${libPath} \
         --add-flags $out/opt/Vesktop/resources/app.asar \
-        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
+        --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
+        ${lib.strings.optionalString withTTS "--add-flags \"--enable-speech-dispatcher\""}
```

Comment:
I feel like the backslash at the end of the Wayland flag is a bit ugly. Perhaps you can move this optional flag between one line up and have the backslash as part of the `optionalString`

Also, a small nitpick: Apparently you should just use `lib.optionalString`. The namespaces within `lib` are just an implementation detail